src/core/solver-base.cpp

```diff
@@ -105,12 +103,92 @@ const double& SolverAbstract::get_xreg() const { return xreg_; }
 
 const double& SolverAbstract::get_ureg() const { return ureg_; }
 
-const double& SolverAbstract::get_stepLength() const { return steplength_; }
+const double& SolverAbstract::get_steplength() const { return steplength_; }
 
 const double& SolverAbstract::get_dV() const { return dV_; }
 
 const double& SolverAbstract::get_dVexp() const { return dVexp_; }
 
+const double& SolverAbstract::get_th_acceptstep() const { return th_acceptstep_; }
+
+const double& SolverAbstract::get_th_stop() const { return th_stop_; }
+
+const std::size_t& SolverAbstract::get_iter() const { return iter_; }
+
+void SolverAbstract::set_xs(const std::vector<Eigen::VectorXd>& xs) {
+  const std::size_t& T = problem_->get_T();
+  if (xs.size() != T + 1) {
+    throw_pretty("Invalid argument: "
+                 << "xs list has to be " + std::to_string(T + 1));
```

Comment:
I think you can do all of this with `<<` and avoid the `to_string` entirely.

Reply:
I think so. I agree with you that it would better to remove std:to_string and it header inclusion.

We have this type of comment in the entire library. I propose to solve it in an dedicated PR. I have opened an issue: https://github.com/loco-3d/crocoddyl/issues/665